### ISSUE TITLE
<input type='file' multiple /> doesn't work on Android devices

### ISSUE THREAD

**Issue by MarksPoint** [OfficeDev/microsoft-teams-library-js#1316]
We've built a message extension that allows users to select files (images) to upload to a channel.

We're using a custom web form that allows a user to select images.
In the form we use ```<input type='file' multiple />``` to let the user select *multiple* images from his/her device at once.

When using the Teams app on Android devices, selecting multiple images doesn't work; only single select is possible.
Using the same web form in the browser (Chrome on Android), selecting multiple images works correctly.
The issue only exists on Android devices, everything works as correctly on iOS.

This is what happens when I use the **Choose files** button from within the Teams app on my Android device. I'm only able to select one image at a time:
![image](https://user-images.githubusercontent.com/16799069/185640597-a70b9ba3-bf0e-42a1-a8e7-b5259b515338.png)

This is what happens when I use the **Choose files** button from within the browser. I'm able to select multiple files at once:
![image](https://user-images.githubusercontent.com/16799069/185641061-939705cc-8a2c-4808-a1f4-9cfb463e6158.png)

## Reproduction input:

To reproduce this issue, I used this sample app:
https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-device-permissions/nodejs

I updated Tab.tsx and added the ```<input type='file' multiple />``` line:
![image](https://user-images.githubusercontent.com/16799069/185641895-de513cda-fd1e-4ab3-8085-e4c5e3991501.png)

This adds a default form field to the page:
![image](https://user-images.githubusercontent.com/16799069/185642484-8e59d4a4-3836-4102-93d2-46566b00743d.png)

I ran the app using ngrok so I was able to use the form both from within the Teams app AND from within a browser.

**It's vital for our application to be able to select multiple images to upload at once.** 

_PS.
Using ```microsoftTeams.media.selectMedia``` as an alternative is not an option because of 2 other issues/limitations: 1) It's possible to select only up to 10 image at a time, which is too limiting for our app. 2) There's a total file size limit of 30MB that makes the image picker break on selecting 10 images with a total size bigger than 30MB. I've posted an issue about this [here](https://github.com/OfficeDev/microsoft-teams-library-js/issues/1317)._

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - We are investigating your query. We will update you soon.

**Comment by MarksPoint** [comment]
Hello @Nivedipa-MSFT, 
Do you have an update regarding my query? It's a very serious issue for us.

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - We are checking this internally. We will update you real soon.

**Comment by MarksPoint** [comment]
I've tested on two devices and the issue is reproducable on both devices:
Android 11 - Teams 1416/1.0.0.2022354201/0815
Android 9 - Teams 1416/1.0.0.2022354201/0815

Could you please share your code with me so I can test with your solution?

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - We have added only following code in our html file.
`<input type="file" id="files" name="files" multiple>`

**Comment by MarksPoint** [comment]
Did you use this sample app:

https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/tab-device-permissions/nodejs

?

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - No. In my custom application I have done changes. Are you using this sample? Can you please share the changes which you have done in this sample?

**Comment by MarksPoint** [comment]
Please have a look at my original post. I mentioned exactly what I did in the section "Reproduction input".

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - We have tested this in Tab device permission sample. We are able to repro your issue. We have raised a bug for this. We will let you know once we have any updates.

**Comment by MarksPoint** [comment]
Thanks for confirming that it's indeed an issue with Teams.
I'm looking forward to a solution and hope you'll keep me posted.

**Comment by MarksPoint** [comment]
I would really love to get an update. Is this something you'll fix soon? Any timelines to share?

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - Engineering team is working on it. Right now, we don't have any ETA to share. We will let you know once we have any update on it.

**Comment by MarksPoint** [comment]
It's been over a month since I last asked about an update and it's been awfully quiet.
What's going on? Is the engineering team still working on it? How are they progressing?

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - We are checking the status of the bug. We will update you soon.

**Comment by MarksPoint** [comment]
It's been over a month since I last asked about an update and it's been awfully quiet.
What's going on? Is the engineering team still working on it? How are they progressing?

I feel like I'm repeating myself, but then again, communication is completely lacking from MS side.

**Comment by ChetanSharma-msft** [comment]
Hello @MarksPoint - I am extremely sorry for delay in responses.
Just wanted to let you know that we are following with engineering team and checking the status of bug fixes.
Once it is resolved, we will definitely inform you.

**Comment by Nivedipa-MSFT** [comment]
@MarksPoint - Apologies for the delay. We are checking the status of bug fix with the engineering teams. We will let you know once we have any update.

**Comment by MarksPoint** [comment]
Again 7 weeks without ANY update. And 4,5 MONTHS since I first reported this issue.

This is frustrating in so many ways:
- In the first place because we're having a serious issue that we have to deal with
- But also because I feel like I'm not taken seriously at all
- And that all the time I spent to report this issue with all the details and all is for nothing

I am very, very disappointed in you, Microsoft, and especially in the team reponsible for support on this repository.

**Comment by ChetanSharma-msft** [comment]
Hello @MarksPoint - We are extremely sorry for delay in response.
Just wanted to inform you that this is currently not supported, and engineering team is working on this feature.

We are actively following with engineering team and let you know the updates, if any.

**Comment by MarksPoint** [comment]
What happened with the classification as "bug"?
> We have raised a bug for this.

Now that it's marked as a feature request I fear that I'll have to wait indefinitely for a fix.

> and engineering team is working on this feature.

Are they really? Because this is what you said on September 8th as well. It's very hard to believe that the engineering team has been workin on a fix for 4 months already. 

Please be honest with me and just tell me that the issue doesn't have any priority on your end and that I shouldn't expect any updates in the near future.

**Comment by ChetanSharma-msft** [comment]
Hello @MarksPoint - We are extremely sorry for the inconvenience.
We are checking with engineering team and let you know as soon as possible.

**Comment by Wajeed-msft** [comment]
Hi @MarksPoint- Platform team recommends using the _selectMedia_ API to upload image files. This supports uploading maximum of 10 files. We have identified an issue in this API which prevents users to upload multiple images in case it overflows the permissible metadata size of 4MB. We have prioritized this issue and investigating it at this point of time. Based on the investigation results, timeline to release the fix will be identified. We should be able to provide you another update in 3 weeks' time as specified in another issue https://github.com/OfficeDev/microsoft-teams-library-js/issues/1317.
